guides/user/canary/stage/index.md

```diff
@@ -0,0 +1,169 @@
+upstream from this stage.
+
+## About the canary stage
+
+The canary analysis can be performed over data points collected beginning from
+the moment of execution and into the future, or it can be performed over a
+specified time interval.
+
+### Real-time versus retrospective analysis
+
+A real-time analysis means that the canary analysis is performed over a time
+interval beginning at the moment of execution. In a retrospective analysis the
+canary analysis is performed over an explicitly specified time interval (likely
+in the past).
+
+### Metric scope
+
+Metric scope is the where and when for your canary analysis. It describes the
+specific baseline and canary server groups, the start and end times and interval,
+and the cloud resource on which the baseline and canary are running.
+
+You can also
+
+
+
+
+## Configure the canary stage
+
+<!-- something here about where in the pipeline you'd put the canary stage, and
+other permutations, like multiple stages? -->
+
+1. In the pipeline in which you will run the canary, click __Add stage__.
+
+   This pipeline needs to be in an application that has access to the [canary
+   configuration](/guides/user/canary/config/) you want to use.
+
+1. For __Type__ select __Canary__.
+
+1. Give the stage a name, and use the __Depends On__ field to position the stage
+downstream of its dependencies.
+
+   ![Canary stage declaration](/guides/user/canary/stage/canary_stage_top.png)
+
+1. Select the __Analysis Type__&mdash;either __Real Time__ or __Retrospective__.
+
+   * __Real Time__
+
+     The analysis happens for a specified time period, beginning when the stage
+     executes.
+
+     For __Real Time__, also specify the number of hours to run (__Lifetime__).
+
+   * __Retrospective__
+
+     Analysis occurs over some specified period. Typically, this is done
+     for a time period in the past, against a baseline deployment and a canary
+     deployment which have already been running before this canary analysis
+     stage starts.
+
+1. Specify the analysis configuration:
+
+   * Choose the __Config Name__.
+
+     This is the canary config you created [here](). That configuration must be
+     created within this same application, unless your Spinnaker is set up so
+     that all canary configurations are available to all pipelines.
+
+   * If you're running a __Real Time__ analysis, set a __Delay__.
+
+     This is the number of minutes to wait before starting the analysis. This
+     gives both the baseline and canary a chance to get warmed up before they're
+     expected to provide meaningful  metrics. Leave blank for a delay of zero
+     minutes.
+
+   * Set a __Delay__.
+
+     For real-time analyses, how many minutes to wait before starting the
+     analysis. The gives both the baseline and canary a chance to get "warmed
+     up" before they're expected to provide meaningful metrics.
+
+   * Set the __Interval__.
```

Comment:
We talked over some examples of how delay, interval, lifetime, and lookback windows could shape how the canary stage works - I think some of those examples could be really helpful in this section.